cirq/contrib/quirk/cells/unsupported_cells.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Iterator
+
+from cirq.contrib.quirk.cells.cell import (
+    CellMaker,
+    CELL_SIZES,
+)
+
+
+def generate_all_unsupported_cell_makers() -> Iterator[CellMaker]:
+    # Post selection.
+    yield from _unsupported_gates(
+        "|0⟩⟨0|",
+        "|1⟩⟨1|",
+        "|+⟩⟨+|",
+        "|-⟩⟨-|",
+        "|X⟩⟨X|",
+        "|/⟩⟨/|",
+        "0",
+        reason='postselection is not implemented in Cirq')
+
+    # Non-physical operations.
+    yield from _unsupported_gates("__error__",
+                                  "__unstable__UniversalNot",
+                                  reason="Unphysical operation.")
```

Comment:
nit: most reasons are not capitalized, so let's s/U/u/ for consistency; also on l.44 below

Reply:
Done.